lib/auth/keystore/manager.go

```diff
@@ -215,6 +219,9 @@ func (opts *Options) CheckAndSetDefaults() error {
 	if opts.Logger == nil {
 		opts.Logger = slog.With(teleport.ComponentKey, "Keystore")
 	}
+	if opts.HealthCallback == nil {
+		opts.HealthCallback = func(_ error) {}
```

Comment:
Why not set `process.onHeartbeatWithRecovery(teleport.ComponentAuthKeystore, 0)` here if the value is not set?

Reply:
the process isn't available in this scope. Also I'm not sure we'd want to pull it in. I think its better to keep this package independent and unaware of those sorts of details.